packages/api/src/@types/custom.d.ts

```diff
@@ -5,3 +5,5 @@ namespace Express {
     email: string | undefined;
   }
 }
+
+declare module "jaro-winkler";
```

Comment:
Already talked about this on the other PR.

Reply:
Sorry! Resolved now.